mcsema/CFG/CFG.h

```diff
@@ -158,6 +159,7 @@ struct NativeXref {
 
   const NativeVariable *var = nullptr;
   const NativeFunction *func = nullptr;
+  const NativeExternalVariable *externalVar = nullptr;
```

Comment:
Can you rename this to `external_var`?

Reply:
Sure, I'll modify the fix according to your comments.